index.html

```diff
@@ -36,6 +36,196 @@ <h1>
             <li><a href="#"><img src="./assets/victory.svg" alt="victory"></a></li>
         </ul>
     </section>
+
+    <section id="works">
+        <div class="project">
+            <div>
+                <img src="./assets/SnapshootPortfolio.svg" alt="">
+            </div>
+
+            <div class="projectDescription">
+                <div class="projectTitle">
+                    <h1>Tonic</h1>
+                </div>
+                <ul class="projectTools">
+                    <li>CANOPY</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>Back End Dev</li>
+                    <li><img src="./assets/Counter.svg" alt=""></li>
+                    <li>2015</li>
+                </ul>
+                <div class="projectSummary">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="projectTechnology">
+                    <span class="tag">HTML</span>
+                    <span class="tag">CSS</span>
+                    <span class="tag">JavaScript</span>
+                </div>
+                <div class="projectDemo">
+                    <button type="button">See project</button>
```

Comment:
- [ ] Good Job in implementing this task, however, As part of the project requirement, Please add the Button interactions in your styles like `enable, hover, etc`, you can check this [link](https://www.w3schools.com/howto/howto_css_animate_buttons.asp) for reference on how to implement it. Below is the Figma design:

![btn Hover](https://github.com/BilalLiaquat7/Portfolio/assets/43528594/e033275d-65bd-4aed-a4c3-00c50bb79387)